docs/software/prgenv/julia.md

```diff
@@ -0,0 +1,50 @@
+[](){#ref-uenv-julia}
+# julia
+
+[Julia](https://julialang.org/) is a programming language that was designed to solve the "two-language problem", the problem that prototypes written in an interactive high-level language like MATLAB, R or Python need to be partly or fully rewritten in lower-level languages like C, C++ or Fortran when a high-performance production code is required. Julia, which has its origins at MIT, can however reach the performance of C, C++ or Fortran despite being high-level and interactive. This is possible thanks to Julia's just-ahead-of-time compilation: code can be executed in an interactive shell as usual for prototyping languages, but functions and code blocks are compiled to machine code right before their first execution instead of being interpreted (note that modules are pre-compiled).
```

Comment:
Please add line breaks between sentences.

Also, I would avoid adding so much background material or advertising up front, so that users can get to the "how do I use the julia uenv" part of the docs.

Reply:
done